internal/builders/gradle/action.yml

```diff
@@ -0,0 +1,127 @@
+# Copyright 2023 SLSA Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Gradle builder
+inputs:
+  # BYOB-provided inputs
+  slsa-workflow-inputs:
+    # Inputs in JSON format.
+    description: 'All the onputs'
+    type: string
+    required: true
+  slsa-layout-file:
+    description: 'Location to store the layout content'
+    type: string
+    required: true
+  slsa-workflow-secret1:
```

Comment:
note: we now support up to 15 secrets: https://github.com/slsa-framework/slsa-github-generator/blob/main/internal/nodejs-action/action.yml#L13-L28
They are required inputs.. even if you don't provide them it won't fail and jut print a warning in the logs. But better add them :)